emulate/emulate.js

```diff
@@ -0,0 +1,103 @@
+const redis = require("redis");
+const redisClient = redis.createClient("redis://redis:6379");
+const { NodeVM } = require("vm2");
+
+// https://www.npmjs.com/package/vm2
+
+redisClient.on("error", function (error) {
+  console.error(error);
+});
+
+/// Handles switching DMX values for the next frame
+function _NextFrame(dmxValuesIn) {
+  // TODO - Check if dmxValuesIn is correct
+  redisClient.set("DMXvalues", dmxValuesIn.toString());
+  redisClient.set("DMXvalues_update_timestamp", Date.now().toString());
+}
+
+/// Handles initializing the array with DMX values
```

Comment:
Ten komentarz nic nie wnosi.
Jeśli dodawać tu komentarz to po to żeby opisać dlaczego to robimy, a nie co robimy.

Czyli np. dlaczego jest [0, 0, 0] (by móc sie odwolywac jak w opencv)
albo
dlaczego 5 rzędów i 28 kolumn? (bo taka jest wielkosc uniwersytu)

Reply:
Na razie usunę zamiast się rozpisywać, później będzie można dodać opis